src/utils/beneficial.owner.gov.ts

```diff
@@ -79,7 +79,7 @@ export const getBeneficialOwnerGovById = (req: Request, res: Response, next: Nex
   }
 };
 
-export const postBeneficialOwnerGov = async (req: Request, res: Response, next: NextFunction, nextPage: string, registrationFlag: boolean): Promise<void> => {
+export const postBeneficialOwnerGov = async (req: Request, res: Response, next: NextFunction, nextPage: string): Promise<void> => {
```

Comment:
You're removing a number of parameter flags here named 'registrationFlag'. Are you sure that's used for save and resume and not to distinguish between a registration and an update/remove journey?

Reply:
it was only being used to pass into the saveAndContinue function, where it was used to check if the registration S&R flag was on, or if registrationFlag was false, would check if Update S&R flag was on